modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -590,52 +612,28 @@ function getIssueFields()
     }
 
     //Now get issue values
-    $issueData = null;
+    $issueData = getIssueData();
     if (!empty($_GET['issueID'])) { //if an existing issue
-        $issueID   = $_GET['issueID'];
-        $issueData = $db->pselectRow(
-            "SELECT i.*, c.PSCID, s.Visit_label as visitLabel FROM issues as i " .
-            "LEFT JOIN candidate c ON (i.candID=c.CandID)" .
-            "LEFT JOIN session s ON (i.sessionID=s.ID) " .
-            "WHERE issueID=:issueID",
-            array('issueID' => $issueID)
-        );
-
-        $isWatching            = $db->pselectOne(
-            "SELECT * FROM issues_watching WHERE 
-                issueID=:issueID AND userID=:userID",
+        $issueID    = $_GET['issueID'];
+        $issueData  = getIssueData($issueID);
+        $isWatching = $db->pselectOne(
+            "SELECT * FROM issues_watching 
```

Comment:
Can this be replaced by an explicit list of fields as long as it's being changed?

It might not be an issue here, but it's best practice for a number of reasons, including: https://blog.jooq.org/2017/03/08/many-sql-performance-problems-stem-from-unnecessary-mandatory-work/